coba/learners/vowpal.py

```diff
@@ -1,22 +1,22 @@
 import re
-
-from sys import platform
 from itertools import repeat, compress
+from sys import platform
 from typing import Any, Dict, Union, Sequence, Mapping, Optional, Tuple
-from coba.backports import Literal
 
+from coba.backports import Literal
 from coba.exceptions import CobaException
-from coba.utilities import PackageChecker
-from coba.primitives import Sparse, Context, Action, Actions
-
 from coba.learners.primitives import Learner, Probs
+from coba.primitives import Sparse, Context, Action, Actions
+from coba.utilities import PackageChecker
 
 Feature       = Union[str,int,float]
 Features      = Union[Feature, Sequence[Feature], Dict[str,Union[int,float]]]
 Namespaces    = Dict[str,Features]
 VW_Features   = Sequence[Union[str,int,Tuple[Union[str,int],Union[int,float]],Dict[str,Union[int,float]]]]
 VW_Namespaces = Dict[str,VW_Features]
 
+DEFAULT_NAMESPACE_INTERACTIONS = (1, 'a', 'ax', 'axx')
```

Comment:
This wasn't a constant but a (mutable) list default argument which is risky, see for example
```
def dummy(mutable_list = [1,2,3]):
    mutable_list.append(0)
    print(mutable_list)
    
dummy()
[1, 2, 3, 0]
dummy()
[1, 2, 3, 0, 0]
dummy()
[1, 2, 3, 0, 0, 0]
```